src/main/java/org/broadinstitute/hellbender/tools/funcotator/FilterFuncotations.java

```diff
@@ -213,10 +231,56 @@ private VariantContext applyFilters(final VariantContext variant, final Set<Stri
         return variantContextBuilder.make();
     }
 
+    private void buildArHetByGene(VariantContext variant) {
+        final Map<Allele, FuncotationMap> funcs = FuncotatorUtils.createAlleleToFuncotationMapFromFuncotationVcfAttribute(
+                funcotationKeys, variant, "Gencode_" + reference.gencodeVersion + "_annotationTranscript", "FAKE_SOURCE");
+
+        funcs.values().forEach(funcotationMap -> {
+            getTranscriptFuncotations(funcotationMap).forEach(funcotations -> {
+                Optional<Map.Entry<String, String>> maybeGeneFuncotation = funcotations.stream().filter(funcotation -> funcotation.getKey().equals("Gencode_27_hugoSymbol")).findFirst();
```

Comment:
It may be worth using a wildcard for the gencode version number here so this could be used on hg19 data as well.

(The hg19 field has a different version number associated with it - 19.)

This would also make it robust to updates to the gencode data source.

----

Actually, there's already a mechanism in the code to do this for now - `reference.gencodeVersion`